apidoc/Titanium/UI/Window.yml

```diff
@@ -439,6 +439,28 @@ properties:
     since: 3.1.3
     osver: {ios: {min: "7.0"}}
 
+  - name: extendSafeArea
+    summary: |
+       Specifies whether the content (subviews) of the window  will render inside the safe-area or not. 
+       Only used in iOS 11 and later.
+    description:  |
+        If the value is `true`, the content (subviews) of the window will not respect the safe-area, meaning that the content 
+        will fill the whole screen available for rendering. If the value is `false`, the content of the
+        window will be rendered inside the safe-area (including then status-bar, iPhone X home-indicator & iPhone X notch).
```

Comment:
@hansemannn  status-bar, iPhone X home-indicator & iPhone X notch are not part of safe area. Let us remove the statement from doc.
And we have planned to make extendSafeArea property false by default to automatically adjust layout in Titanium SDK 7.0.0. Do you really think we need to change it. In my view default property should be true. So that when developer change this property they will be aware of consequences of property and will change this wherever they require and verify the same. What you think ?

Let us discuss on this. Thanks.